assets/scripts/evolution/svg_pagination.js

```diff
@@ -0,0 +1,16 @@
+export default function() {
+	(function() {
+		var elems = document.querySelectorAll('ul.evo_c-svg-dot--default li'),
+		makeActive;
+
+		makeActive = function() {
```

Comment:
Hello @jmatembu , after I finished the modification, so I just need to push again or create another pull request? Thanks!